src/wmtk/attribute/MutableAccessor.hpp

```diff
@@ -1,67 +1,8 @@
 #pragma once
 #include "ConstAccessor.hpp"
 
-
-namespace wmtk {
-class Mesh;
-class TriMesh;
-class TetMesh;
-class EdgeMesh;
-class PointMesh;
-class TriMeshOperationExecutor;
-} // namespace wmtk
 namespace wmtk::attribute {
-
-/**
- * Same as ConstAccessor but with the ability to write to the attributes.
- */
 template <typename T>
-class MutableAccessor : public ConstAccessor<T>
-{
-public:
-    friend class wmtk::Mesh;
-    friend class wmtk::TetMesh;
-    friend class wmtk::TriMesh;
-    friend class wmtk::EdgeMesh;
-    friend class wmtk::PointMesh;
-    friend class wmtk::TriMeshOperationExecutor;
-    using CachingBaseType = CachingAccessor<T>;
-    using ConstAccessorType = ConstAccessor<T>;
-
-    using ConstAccessorType::ConstAccessorType;
-
-    using ConstAccessorType::const_scalar_attribute;
-    using ConstAccessorType::const_vector_attribute;
-    using ConstAccessorType::dimension;
-    using ConstAccessorType::reserved_size;
-    using ConstAccessorType::scalar_attribute;
-    using ConstAccessorType::vector_attribute;
-
+using MutableAccessor = TupleAccessor<T>;
```

Comment:
I think `MutableAccessor` can be removed completely.

Reply:
i thought i'd deleted this file :upside_down_face: . wilco